Building with closures and this keyword/index.html

```diff
@@ -0,0 +1,15 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+          name="viewport">
+    <meta content="ie=edge" http-equiv="X-UA-Compatible">
+    <title>Document</title>
+</head>
+<body>
+<button id="click-me-btn">click me</button>
+
+<script src="main.js"></script>
```

Comment:
add defer  'keyword' to your script tag and avoid placing it in the body.

`<script src="" defer></script>`